R/connection-pane.R

```diff
@@ -841,12 +916,14 @@ preview_object <- function(host, token, rowLimit,
 #' open_workspace(host = db_host(), token = db_token, name = "MyWorkspace")
 #' }
 #' @importFrom glue glue
-open_workspace <- function(host = db_host(), token = db_token(), name = NULL) {
+open_workspace <- function(host = db_host(), token = db_token(), name = db_wsid()) {
```

Comment:
Fixes the name  for Connections tab

Reply:
I'd be open to making db_host() the default for `name` - that seems very sensible!

Reply:
Id's tend to be tricky for people to remember which is which

Reply:
I agree, although the db_host(0 is equally confusing for me.  Maybe introducing workspace alias (which would be up to user to define in .databrickscfg) is the way to go. There user could define **test-ws**, **prod-ws** etc, something that is readable.